### PR TITLE
Track market invalid bids

### PR DESCRIPTION
**Pull request by bthaile**
https://github.com/AugurProject/augur/issues/8525

SDK portion of market invalid bids. 
created subscription to spit out Invalid bids to populate UI Invalid outcome in orderbook. sending all Bids so there is no management with regular orderbook data when user nav to market trading page.

`trackMarketInvalidBids` config, we might want to toggle it off for sportsbook.

### CODE REVIEW

File: packages/augur-sdk/src/api/MarketInvalidBids.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    );
+  }
+
+  getMarketInvalidBids(orders: { logs: ParsedOrderEventLog[] }) {
+    const onlyInvalidBids = orders.logs.filter(
+      (o) => String(o.orderType) === OrderTypeHex.Bid && o.outcome === `0x00`
+    );
+    const markets = _.map(onlyInvalidBids, 'market');
+    const marketIds = Array.from(new Set(markets));
+    Promise.all(
```
Comment (pgebheim) [inline packages/augur-sdk/src/api/MarketInvalidBids.ts:27]: I think promise.all actually had a limit to the number of promises it handles..

@justinbarry was this some platform specific thing?

**Review by pgebheim — APPROVED** [review]
Approving but we need to check the limits of Promise.all

**Comment by bthaile** [comment]
@Mergifyio update

**Comment by pgebheim** [comment]
@bthaile this needs to go into dev as well yes?

**Pull request merged**